packages/scanner-global-library/src/page.ts

```diff
@@ -33,8 +38,13 @@ export class Page {
         @inject(GlobalLogger) @optional() private readonly logger: GlobalLogger,
     ) {}
 
-    public async create(browserExecutablePath?: string): Promise<void> {
-        this.browser = await this.webDriver.launch(browserExecutablePath);
+    public async create(options?: BrowserStartOptions): Promise<void> {
+        if (options?.browserWSEndpoint !== undefined) {
+            this.browser = await this.webDriver.connect(options.browserWSEndpoint);
```

Comment:
can we do the fall back here to the browser executable if it fails to launch

Reply:
discussed offline, we plan to have the fallback logic in the web-api-scan-runner so each scan entry point can make the decision independently